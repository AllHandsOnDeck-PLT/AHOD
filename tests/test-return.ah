int w
string x
bool y 
float z 
when do int A():
{
  return 3
}
when do string B():
{
  return "some plt fun"
}
when do bool C():
{
  return true
}
when do float D():
{
  return 3.5
}
main:
{
  w = do A() 
  do PRINT(w)
  x = do B() 
  do PRINT(x)
  y = do C() 
  do PRINT(y)
  z = do D() 
  do PRINT(z)
}
