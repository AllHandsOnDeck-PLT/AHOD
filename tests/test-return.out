3
some plt fun
1
3.5